src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -299,7 +294,7 @@ def generate_MCP_1_default_config(self, filename):
                                 'key': os.path.join('certs', 'domain_proxy.key')},
                                {'cert': os.path.join('certs', 'domain_proxy_1.cert'),
                                 'key': os.path.join('certs', 'domain_proxy_1.key')}],
-        'deltaIap': 2
+        'deltaIap': 1
```

Comment:
I think we don't need this anymore, since you have IAP_MARGIN above. If you want to keep it, please at least set it to that value. (Assuming my guess about the meaning of IAP_MARGIN is correct.)

Reply:
Addressed as per the comment